src/bindings/python/src/openvino/runtime/__init__.py

```diff
@@ -5,6 +5,27 @@
 """openvino module namespace, exposing factory functions for all ops and other classes."""
 # noqa: F401
 
+import warnings
+warnings.simplefilter("always", DeprecationWarning)
+import inspect
+stack = inspect.stack()
```

Comment:
do we really need this stack? Can it degrade performance in case import `openvino.runtime` from different modules?
May be it is sufficient just warns:
```
    warnings.warn(
        "The `openvino.runtime` module is deprecated and will be removed in the 2026.0 release. "
        "Please replace `openvino.runtime` with `openvino`.",
        DeprecationWarning,
        stacklevel=2
    )
```
For us, it will be easy to remove usage of `openvino.runtime` without any stack. For whom, this stack will be helpful?

Reply:
`inspect.stack()` will be called on `openvino.runtime` initialization. If there is one python proccess it should be called once afaik. 
I created the version with inspect.stack() to help users locate in their code what triggers the warning. Such information is usually provided by `warnings` package, but here it will be lost because of `LazyLoader`.
Lazy loading is needed because we don't want to load `openvino.runtime` before someone actually uses it (actual use of the `openvino.runtime`  triggers warning).

Reply:
For users, it is easy to locate `openvino.runtime` by just simple searching. Moreover, using `inspect.stack` can degrade performance and memory usage. I am leaning to have simple warning.

Reply:
I updated to have one version of warning.